### PR TITLE
fixed error where the control state display buttons are selectable

### PR DESCRIPTION
**Pull request by andrewhong04**

### CODE REVIEW

File: tuxemon/states/control/__init__.py [hunk 1]
```diff
@@ -165,8 +165,7 @@ def change_state(
             label = f"{T.translate(key).upper()}"
             image = self.shadow_text(label)
             item = MenuItem(image, label, None, change_state("SetKeyState", input=input))
-            if input is None:
-                item.enabled = False
+            item.enabled = item is not None
```
Comment (bitcraft) [inline tuxemon/states/control/__init__.py]: the name "input" is reserved for python, and its bad practice to "shadow" an existing global or builtin name ("input").  i would say that replaceing "input", with either "button", or "button_name" on line 164 would be best here.
```python
item.enabled = button is not None
```
(change "button" to whatever you rename "input")

**Review by bitcraft — CHANGES_REQUESTED** [review]

File: tuxemon/states/control/__init__.py [hunk 2]
```diff
@@ -161,11 +161,11 @@ def change_state(
             (display_buttons[buttons.BACK], None)
         )
 
-        for key, input in key_items_map:
+        for key, button in key_items_map:
             label = f"{T.translate(key).upper()}"
             image = self.shadow_text(label)
-            item = MenuItem(image, label, None, change_state("SetKeyState", input=input))
-            item.enabled = item is not None
+            item = MenuItem(image, label, None, change_state("SetKeyState", button=button))
```
Comment (bitcraft) [inline tuxemon/states/control/__init__.py:167]: should be `input=button`, unless you want to refactor the SetKeyState

Comment (andrewhong04) [reply tuxemon/states/control/__init__.py:167]: Yeah, I already did that in line 65. Is there an issue if I refactor it?

Comment (bitcraft) [reply tuxemon/states/control/__init__.py:167]: i see that now!  no problem then

**Review by bitcraft — APPROVED** [review]
thank you!

**Pull request merged**